### PR TITLE
Fix typos in "Signals" documentation section

### PR DESCRIPTION
**Pull request by fabrideci**
This PR includes the code required to fix two simple typos in the "Logs" and "Traces" pages of the "Concept - Signals" documentation section.

### CODE REVIEW

File: content/en/docs/concepts/signals/traces.md [hunk 1]
```diff
@@ -298,7 +298,7 @@ Each span has a status. The three possible values are:
 The default value is `Unset`. A span status that is `Unset` means that the
 operation it tracked successfully completed without an error.
 
-When a span status is `Error`, then that means some error ocurred in the
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:301]: @svrnm - we should find out why cSpell doesn't detect this. I made some quick tests but wasn't able to find out why. Maybe a bug with the its dictionary?

**Review by chalin — APPROVED** [review]
Thanks for the fixes @fabrideci. We'll need you to sign the CLA before we can merge this.

File: content/en/docs/concepts/signals/traces.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/concepts/signals/traces.md:301]: that's odd indeed, they even have it here:

https://github.com/streetsidesoftware/cspell/blob/3914546fdd862809df2ebf466e502adfbfd93384/rfc/rfc-0001%20suggestions/fixtures/EnglishTypos_common.typo.txt#L2584

**Review by theletterf — APPROVED** [review]

File: content/en/docs/concepts/signals/traces.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/concepts/signals/traces.md:301]: https://github.com/streetsidesoftware/cspell-dicts/blob/dc56f6f63e68134c012456f796ef5f67d745ff06/dictionaries/en_US/src/en_US.txt#L13385 ... ❓

Comment (svrnm) [reply content/en/docs/concepts/signals/traces.md:301]: raised an issue: https://github.com/streetsidesoftware/cspell-dicts/issues/2940

**Review by cartermp — APPROVED** [review]

**Pull request merged**